### PR TITLE
Added a dark theme to the style.css

### PR DESCRIPTION
**Pull request by JaidenW**

### CODE REVIEW

**Comment by JaidenW** [comment]
<footer class="text-center text-muted fixed-bottom bg-light"> Should remain the same, i didn't realize that it was fixed and overlaps some elements, the bg-light tag is required, & handled via CSS override :)

File: app/static/style.css [hunk 1]
```diff
@@ -10,3 +10,55 @@ header h1 {
   font-weight: 700;
   text-decoration: none;
 }
+
+/* Super simple dark theme - By Jaiden W */
+
+body.dark-theme{
+  background:#202020;
+  color:#fff!;
```
Comment (nitnelave) [inline app/static/style.css:18]: I think we don't want pure white on pure black. If we take Material design's recommendations (not that it's gospel, but it's a good starting guideline) we'd have a background of #121212 for the very back, up to #242424 for the most accentuated background, and a text color of at most #dedede

**Review by nitnelave — CHANGES_REQUESTED** [review]

**Comment by nitnelave** [comment]
What would you think of something like https://vinorodrigues.github.io/bootstrap-dark/ ? Rather than creating/maintaining our own dark theme.

**Comment by JaidenW** [comment]
That might be better, especially if you intend to add anything else to it in the future, that would likely also likely make it easier for you to style things on your own

**Comment by nitnelave** [comment]
Closing in favor of https://github.com/nitnelave/lldap/issues/250

**Pull request closed without merging**